docs/modules/ROOT/nav.adoc

```diff
@@ -15,6 +15,7 @@
 ** xref:redis-store.adoc[Redis Store]
 ** xref:chroma-store.adoc[Chroma Store]
 ** xref:pinecone-store.adoc[Pinecone Store]
+** xref:pgvector-store.adoc[PgVector (Postgres) Store]
```

Comment:
Isn't it PostgreSQL ? See https://www.postgresql.org/

Reply:
YES, please! Let's not get the people using `Postgres` win.

Reply:
Oh, good old battles over petty stuff :)